apps/backend/src/app/controllers/run/run.resolver.ts

```diff
@@ -1,8 +1,32 @@
-import { Resolver } from "@nestjs/graphql";
-// import { Run as GQLRun } from "../../types/graphql";
+import { Resolver, Mutation, Args } from "@nestjs/graphql";
 import { RunService } from "./run.service";
 
 @Resolver("Run")
 export class RunResolver {
```

Comment:
If only logged in users can access these, then please put `@UseGuards(FitbitGuard)` at the top